designcompose/src/main/java/com/android/designcompose/DesignText.kt

```diff
@@ -364,6 +384,29 @@ internal fun DesignText(
                 )
             } else {
                 var textLayoutResult: TextLayoutResult? = null
+                val uriHandler = LocalUriHandler.current
+                if (annotatedText.getUrlAnnotations(0, annotatedText.text.length).isNotEmpty()) {
+                    val onClick: (Int) -> Unit = { offset ->
+                        annotatedText
+                            .getUrlAnnotations(start = offset, end = offset)
```

Comment:
So if the hyperlink is at (10, 20) and you click at position 15, `annotatedText.getUrlAnnotation(15, 15)` will return the annotation with the hyperlink?

Reply:
Yes. That is right.